felix/bpf/ut/pol_prog_test.go

```diff
@@ -225,10 +222,19 @@ func icmpPktWithTypeCode(src, dst string, icmpType, icmpCode int) packet {
 }
 
 func packetNoPorts(proto int, src, dst string) packet {
+	// Just using ResolveUDPAddr to parse a string to ip and port
+	srcAddr := net.ParseIP(src)
+	if srcAddr == nil {
+		panic(fmt.Errorf("failed to parse src addr %v", src))
```

Comment:
Should we rather use like `Expected` in test rather then if then panic?

Reply:
Ideally, but that need changes which prevents using `packetNoPorts` in the list of test cases as it is now.